### DISCUSSION TITLE
权限分配问题

### DISCUSSION THREAD

**Discussion by vlinLi** [zuihou/zuihou-admin-cloud#108] [02.提问]
1. lamp_web权限分配问题，取消勾选，提交后刷新任然显示并勾选

![image](https://user-images.githubusercontent.com/49149431/134218212-2f661a1f-457d-4521-bc9d-095b5ace8c59.png)

**Comment by zuihou** [comment]
bug 

https://github.com/zuihou/lamp-cloud/issues/110
